nav-challenge/src/components/formFieldSet/FormFieldSet.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from "react";
+
+interface FormFieldSetProps {
+  type: string;
+  label: string;
+  inputValue: string;
+  placeHolder: string;
+  handleChange: (e: any) => void;
+}
+
+const FormFieldSet: FC<FormFieldSetProps> = ({
+  type,
+  label,
+  inputValue,
+  placeHolder,
+  handleChange,
+}) => {
+  return (
+    <fieldset>
+      <label>{label}</label>
+      <input
+        type={type}
+        placeholder={placeHolder}
+        className="form-control"
+        value={inputValue}
+        onChange={e => {
+          handleChange(e);
```

Comment:
If `handleChange` takes only one parameter and `onChange` expects a callback with one parameter we can write this shorter and cleaner like `onChange={handleChange}`